app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Open Event Organizer</string>
+    <string name="app_name">eventyay organizer</string>
```

Comment:
@mejariamol @iamareebjamal please do not hard code the app name. Make it configurable during the build process.